R/viz.R

```diff
@@ -1958,8 +1959,8 @@ prettyCoOncoplot = function(maf,
                             comparison_column,
                             comparison_values,
                             label1,
-                            label2,
-                            ...){
```

Comment:
Ah I see, I removed this since `devtools::check` complained about this "parameter" not being described. I will revert this and make the changes necessary.